rs/nervous_system/linear_map/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+use rust_decimal::Decimal;
+use std::ops::Range;
+
+// DO NOT MERGE - Migrate SNS rewards to this.
+/// A function that maps from one interval to another.
+#[derive(Clone, Eq, PartialEq, Debug)]
+pub struct LinearMap {
```

Comment:
Can you add some documentation?  I don't really understand the intended context or application of this structure.  After looking at the tests, I understand what it does, but I'm left with the question "Why do I need this?"

The "Why is this needed?  When is this useful?" becomes harder to answer when it's in its own library, b/c now it's pulled out of the context that justifies it.

Reply:
Did you see this triple slash comment? I thought that would be sufficient. I can certainly elaborate (tomorrow).

Reply:
Done.